py-wonnx/src/lib.rs

```diff
@@ -0,0 +1,43 @@
+use pyo3::prelude::*;
+use pyo3::types::PyDict;
+use std::collections::HashMap;
+
+use wonnx::Session;
+#[pyclass]
+#[repr(transparent)]
+pub struct PySession {
```

Comment:
Could make this class name simply 'Session' in Python by using `#[pyclass(name="Session")]` (see [here](https://pyo3.rs/master/class.html#customizing-the-class)) :-)

Reply:
Yeah, the thing is I may wrap PySession within a Python Class called Session, to add other functionality, sort of like in polars: https://github.com/pola-rs/polars/blob/master/py-polars/polars/internals/frame.py

Thanks for the suggestion though. Next time I'll run the PR by you.